tests/tensorlake/testing.py

```diff
@@ -25,3 +27,11 @@ def test_graph_name(test_case: unittest.TestCase) -> str:
     ...         # test_graph_reduce_test_simple
     """
     return unittest.TestCase.id(test_case).replace(".", "_")
+
+
+def wait_function_output(graph: RemoteGraph, invocation_id: str, func_name: str) -> Any:
```

Comment:
There is also a new wait API endpoint exposed by the server.

Reply:
This is nice